WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemProgressBar.kt

```diff
@@ -0,0 +1,7 @@
+package org.wordpress.android.viewmodel.posts
+
+sealed class PostListItemProgressBar(val visibility: Boolean) {
```

Comment:
We might want to consider renaming this class to ProgressBarUiState and moving it into a different package. It seems it can be reused for any feature. Wdyt?

Reply:
Yes, thanks.

Reply:
Resolved in 09ddc74

Reply:
All the variables now contain word "state" twice - eg. `progressBarUiStateState`. Is that intentional?

Reply:
No, it wasn't. It occurred during a refactor of the name and I fixed it in a newer commit. Thanks for spotting nonetheless. You have a keen eye 😉